components/table_kv/src/obkv.rs

```diff
@@ -501,6 +510,30 @@ impl TableKv for ObkvImpl {
         Ok(values.remove(VALUE_COLUMN_NAME).map(Value::as_bytes))
     }
 
+    fn get_batch(&self, table_name: &str, keys: Vec<&[u8]>) -> Result<Vec<Option<Vec<u8>>>> {
+        let mut batch_ops = ObTableBatchOperation::with_ops_num_raw(keys.len());
+        let mut batch_res = Vec::with_capacity(keys.len());
+
+        for key in keys {
+            batch_ops.get(bytes_to_values(key), vec![VALUE_COLUMN_NAME.to_string()]);
+        }
+
+        let result = self
+            .client
+            .execute_batch(table_name, batch_ops)
+            .context(GetBatchValue { table_name })?;
+
+        for table_ops_result in result {
+            match table_ops_result {
+                TableOpResult::RetrieveRows(mut values) => {
+                    batch_res.push(values.remove(VALUE_COLUMN_NAME).map(Value::as_bytes))
+                }
+                TableOpResult::AffectedRows(_) => {}
```

Comment:
Maybe we should return error if this branch is entered.

Reply:
Done